packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -77,6 +77,28 @@ const AlarmColPool = connectStore(() => ({
   return <Link to={`pools/${pool.id}`}>{pool.name_label}</Link>
 })
 
+const DUPLICATE_MAC_ADDRESS = [
+  {
+    name: _('vifMacLabel'),
+    itemRenderer: vif => vif.MAC,
+    sortCriteria: vif => vif.MAC,
+  },
+  {
+    name: _('vm'),
+    itemRenderer: vif => (
+      <Link to={`vms/${vif.$VM}`}>
+        <Vm id={vif.$VM} />
+      </Link>
```

Comment:
For each MAC address, we want to show all the VIFs that have this MAC address (and their VMs next to them) in the same table row.
```
| <MAC1> | <VIF1> on <VM1> (<network1>) |
|        | <VIF2> on <VM2> (<network2>) |
|--------|------------------------------|
| <MAC2> |              ...             |
```